src/app/search/map-friendly-values.pipe.ts

```diff
@@ -103,7 +103,9 @@ export class MapFriendlyValuesPipe implements PipeTransform {
         [SourceFile.TypeEnum.DOCKSTOREYML, 'Configuration'],
         [SourceFile.TypeEnum.DOCKSTORESERVICEOTHER, 'Service Files']
       ])
-    ]
+    ],
+    ['success', new Map([['true', 'Success'], ['false', 'Failed']])],
+    ['type', new Map([['PUSH', 'Git Push'], ['DELETE', 'Git Delete'], ['INSTALL', 'Install']])]
```

Comment:
Discussion on Slack may override this comment, but I would just say Push and Delete instead of Git Push and Git Delete.

You want the most significant word to be on the left (in a left-to-right language, which we are). If I'm scanning the column, I have to read past the "Git" to get to the interesting part of the data, the push or the delete.

Reply:
I agree, I like simply Push and Delete and Install.

Reply:
Done.